File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/felles/Underenhet.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Builder
@Data
@AllArgsConstructor
public class Underenhet implements Virksomhet {
  private final Orgnr orgnr;
  private final Orgnr overordnetEnhetOrgnr;
  private final String navn;
  private final Næringskode5Siffer næringskode;
  private final int antallAnsatte;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/felles/Underenhet.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles

data class Underenhet(
    override var orgnr: Orgnr,
    var overordnetEnhetOrgnr: Orgnr? = null,
    override var navn: String? = null,
    override var næringskode: Næringskode5Siffer? = null,
    var antallAnsatte: Int? = null,
) : Virksomhet